src/status_im/data_store/networks.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im.data-store.networks
+  (:require [status-im.data-store.realm.networks :as data-store])
+  (:refer-clojure :exclude [exists?]))
```

Comment:
Seems like exists? is namespaced, is this necessary then?

Reply:
yeah, true. That's a legacy, don't forget this PR is on its 4th month now :D